### PR TITLE
Translates - French

### PR DESCRIPTION
**Pull request by Alfred-Neuman**
Translate French

Good repo this time ;)

The deletions are not mine. 
It's after a repo update. 
These are just indentations

### CODE REVIEW

**Comment by Silence-112** [comment]
In future, use the unstable branch for your PRs. Stable is back at 10.5.0, unstable is currently at ~10.6.4. Helps avoid merge conflicts

File: A3A/addons/maps/Stringtable.xml [hunk 1]
```diff
@@ -444,101 +498,122 @@
...
                 <Russian>Логгировать использование дебаг консоли</Russian>
                 <Korean>설정 중 디버그 콘솔 사용 기록하기</Korean>
+                <French>Utilisation de la console de débogage pendant l'installation</French>
             </Key>
             <Key ID="STR_antistasi_mission_info_journal_A3A_logDebugConsole_None">
                 <Original>None</Original>
                 <Russian>Не логировать никого</Russian>
                 <Korean>없음</Korean>
+                <French>N'enregistrez personne
+</French>
```
Comment (Silence-112) [inline A3A/addons/maps/Stringtable.xml]:
```suggestion
                <French>N'enregistrez personne</French>
```

Comment (Alfred-Neuman) [reply A3A/addons/maps/Stringtable.xml]: thank you, it's not good to work at night

File: A3A/addons/scrt/Stringtable.xml [hunk 2]
```diff
@@ -2957,636 +2957,742 @@
...
                 <Chinesesimp>空中快速反应部队</Chinesesimp>
                 <Korean>공수부대 신속대응군</Korean>
+                <French>QRF aérienne</French>
             </Key>
             <Key ID="STR_notifiers_SUP_execute_QRFAirdrop">
                 <Original>A %1 air mechanized QRF just arrived</Original>
                 <Russian>Десантные механизированные силы быстрого реагирования %1 прибыли</Russian>
                 <Chinesesimp>%1 空中机械化快速反应部队到达</Chinesesimp>
                 <Korean>%1의 공수 기계화 신속대응군이 방금 도착했습니다</Korean>
+                <French></French>
```
Comment (Silence-112) [inline A3A/addons/scrt/Stringtable.xml]: Is this empty for a reason?

Comment (Silence-112) [reply A3A/addons/scrt/Stringtable.xml]: Seems to be quite a few empty ones in that file

Comment (Alfred-Neuman) [reply A3A/addons/scrt/Stringtable.xml]: It was to remind me to review the documents and correct some translations

**Comment by Alfred-Neuman** [comment]
Hello

I didn't add a translation today.
I must have made a mistake.

very sorry for the conflit

**Comment by Silence-112** [comment]
Conflict is resolved, they tend to happen when working with a lot of files that change somewhat frequently

**Review by Silence-112 — COMMENTED** [review]
Looks generally fine, will do some testing when possible

**Review by Silence-112 — APPROVED** [review]
Looks fine, no errors on startup or mission init

**Pull request merged**